build/ci/evergreen.yml

```diff
@@ -247,7 +247,7 @@ functions:
     - command: shell.exec
       type: setup
       params:
-        working_dir: src/github.com/mongodb/mongocli
```

Comment:
adding `macos-notary.zip` to the gitignore is probablya better fix

Reply:
ack. I will do it